src/main.js

```diff
@@ -187,15 +203,13 @@ function startCountDown() {
       seconds < 10 ?  displaySec.innerText = `0${seconds}` : displaySec.innerText = seconds;
```

Comment:
The assignment on this line is pretty wet. 

In general, take a look at the ternary syntax and how it is used in conjunction with variable assignment.